wiki/Performance_Troubleshooting/en.md

```diff
@@ -0,0 +1,88 @@
+
+### Input latency
+
+There is a noticeable delay between tapping on the keyboard or moving your mouse or tablet pen, and seeing a resulting action on the screen. When diagnosing or reporting, it may help to:
+
+- Check whether it happens on all input devices, or one specifically.
+- If you are using non-official drivers, try switching to official ones instead.
+- Make sure you are running osu! in exclusive fullscreen mode. Running windowed or borderless adds up to one frame of latency.
+
+### Loss of control
+
+During gameplay, one or more input devices stop responding. Everything on screen moves correctly but it’s not possible to play properly until control returns. When diagnosing or reporting, it may help to:
+
+- Press Ctrl-F11 to see the frame time display graph. Check whether there is a lot of white on the graph when this occurs (this means another app or driver on your system is using too much CPU time, delaying input).
+
+### Low frame rate
+
+During gameplay, the frame rate is unable to keep up, resulting in jittery and awkward movement of objects on the screen. This can also result in higher perceived input latency. When diagnosing or reporting, it may help to:
+
+- Check whether it still happens straight after a PC restart.
+- Try closing any application which shows overlays on the game (Discord, Steam, etc.).
+- Make sure you're not forcing vsync on at your display drivers (it should be set to "off" or "application preference").
+- Try changing frame limiter and see if it improves the situation.
+- Make sure you are running osu! in exclusive (not borderless) fullscreen mode. This will always improve performance.
+- Make sure you have [game mode](https://www.windowscentral.com/how-enable-disable-game-mode-windows-10) enabled. It allows windows to provide dedicated cores and resources to osu!.
+- Open task manager and order the list (under Details) by CPU. Look for any high CPU usage during periods of low frame rate, for instance by a background virus or malware scanner. You can run osu! windowed or leave task manager on a second display to diagnose this while you are playing / running a beatmap in autoplay.
+
+## Frame Limiters
+
+osu! offers multiple choices of frame limiters. Each of these have pros and cons. Note that regardless of setting, osu! will limit frame rate at menus to conserve on CPU usage.
+
+### VSync
+
+This frame limiter relies on a driver provided vertical synchronising mechanism. It guarantees no screen tearing, but adds 1-2 frames worth of latency due to the requirement of buffering frames before display.
+
+Generally not recommended for 60hz displays. It can become more useful on 120hz+, but with GSync and FreeSync monitors, the requirement to use VSync to avoid tearing is lifted.
+
+### Power Saving
+
+This frame limiter aims to save on power usage while providing a consistent game experience. Will run at 2x monitor refresh rate and attempt to keep a consistent frame pacing.
```

Comment:
```suggestion
This frame limiter aims to save on power usage while providing a consistent game experience. It will run at 2x monitor refresh rate and attempt to keep a consistent frame pacing.
```